rfcs/RangeSlider.md

```diff
@@ -0,0 +1,62 @@
+# RFC: Add RangeSlider option to existing Slider component
```

Comment:
Small thing but can you move this into a `components` sub-folder?

Reply:
Done